### ISSUE TITLE
_utils.py doesn't find tkimg/linux64 files

### ISSUE THREAD

**Issue by Akuli** [RedFantom/ttkthemes#17]
I cloned the repository and tried to use it like this:

```
$ cd /tmp
$ git clone https://github.com/RedFantom/ttkthemes
$ cd ttkthemes
$ git checkout tree
```

```python
Python 3.4.3 (default, Nov 28 2017, 16:41:13) 
[GCC 4.8.4] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ttkthemes
>>> ttkthemes.ThemedStyle()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/ttkthemes/ttkthemes/themed_style.py", line 29, in __init__
    ThemedWidget.__init__(self, self.tk)
  File "/tmp/ttkthemes/ttkthemes/_widget.py", line 46, in __init__
    self._img_support = self._load_tkimg()
  File "/tmp/ttkthemes/ttkthemes/_widget.py", line 209, in _load_tkimg
    raise RuntimeError("Unsupported platform and/or architecture")
RuntimeError: Unsupported platform and/or architecture
>>> ttkthemes._utils.get_tkimg_directory()
'/tmp/ttkthemes/ttkthemes/tkimg/linux64'
>>> os.path.exists('/tmp/ttkthemes/ttkthemes/tkimg/linux64')
False
>>> os.path.exists('/tmp/ttkthemes/tkimg/linux64')
True
```

**Comment by RedFantom** [comment]
I have tested the fix in `d02da71`, but could you test it as well to make sure it works as intended, @Akuli ?

**Comment by Akuli** [comment]
Works for me.
